### PR TITLE
Added keymaps for Fourier, Levinson

### PR DESCRIPTION
**Pull request by Valgrahf**
Added keymaps for both Levinson and Fourier keyboard with RGB backlighting.

### CODE REVIEW

File: keyboards/fourier/config.h [hunk 1]
```diff
@@ -24,5 +24,11 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #ifdef SUBPROJECT_rev1
     #include "rev1/config.h"
 #endif
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/config.h]: Merge Conflict

File: keyboards/fourier/fourier.h [hunk 2]
```diff
@@ -12,12 +12,20 @@
     LA1, LA2, LA3, LA4, LA5, LA6, RA1, RA2, RA3, RA4, RA5, RA6, RA7, \
     LB1, LB2, LB3, LB4, LB5, LB6, RB1, RB2, RB3, RB4, RB5,      RB7, \
     LC1, LC2, LC3, LC4, LC5, LC6, RC1,      RC3, RC4, RC5, RC6, RC7, \
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/fourier.h]: Merge conflict

File: keyboards/fourier/fourier.h [hunk 3]
```diff
@@ -12,12 +12,20 @@
...
+    LD1, LD2, LD3,     LD5,       RD1,      RD4, RD5, RD6, RD7 \
+=======
     LD1, LD2, LD3, LD4, LD5,      RD1,      RD4, RD5, RD6, RD7 \
+>>>>>>> 19a1fbaca2137a9d4cd529e363039a612a4f941d
     ) \
     KEYMAP( \
         KC_##LA1, KC_##LA2, KC_##LA3, KC_##LA4, KC_##LA5, KC_##LA6, KC_##RA1, KC_##RA2, KC_##RA3, KC_##RA4, KC_##RA5, KC_##RA6, KC_##RA7, \
         KC_##LB1, KC_##LB2, KC_##LB3, KC_##LB4, KC_##LB5, KC_##LB6, KC_##RB1, KC_##RB2, KC_##RB3, KC_##RB4, KC_##RB5,           KC_##RB7, \
         KC_##LC1, KC_##LC2, KC_##LC3, KC_##LC4, KC_##LC5, KC_##LC6, KC_##RC1,           KC_##RC3, KC_##RC4, KC_##RC5, KC_##RC6, KC_##RC7, \
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/fourier.h]: merge conflict

File: keyboards/fourier/keymaps/default/keymap.c [hunk 4]
```diff
@@ -19,12 +19,18 @@ enum custom_keycodes {
 #define KC_ KC_TRNS
 #define _______ KC_TRNS
 #define XXXXXXX KC_NO
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/keymaps/default/keymap.c]: Merge conflict

File: keyboards/fourier/keymaps/default/keymap.c [hunk 5]
```diff
@@ -39,6 +45,16 @@ enum custom_keycodes {
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
   [_BASE] = KC_KEYMAP(
  //,----+----+----+----+----+----|----+----+----+----+----+----+----.
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/keymaps/default/keymap.c]: merge conflict

File: keyboards/fourier/keymaps/default/keymap.c [hunk 6]
```diff
@@ -47,30 +63,49 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
     LCTL ,LALT,LGUI ,FN1 , SPFN1  ,  BSFN2 ,RGUI ,RALT , FN2 , RCTL 
  //`-----+----+-----+----+--------|--------+-----+-----+-----+------'
+>>>>>>> 19a1fbaca2137a9d4cd529e363039a612a4f941d
   ),
 
   [_FN1] = KC_KEYMAP(
  //,----+----+----+----+----+----|----+----+----+----+----+----+----.
     GRV , 1  , 2  , 3  , 4  , 5  , 6  , 7  , 8  , 9  , 0  ,MINS,EQL ,
  //|----`----`----`----`----`----|----`----`----`----`----`----`----|
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/keymaps/default/keymap.c]: Merge Conflict

File: keyboards/fourier/keymaps/default/keymap.c [hunk 7]
```diff
@@ -47,30 +63,49 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
     RTOG ,    ,     ,    ,        ,  DEL   ,  0  ,LEFT ,DOWN , RGHT 
  //`-----+----+-----+----+--------|--------+-----+-----+-----+------'
+>>>>>>> 19a1fbaca2137a9d4cd529e363039a612a4f941d
   ),
 
   [_FN2] = KC_KEYMAP(
  //,----+----+----+----+----+----|----+----+----+----+----+----+----.
     TILD,EXLM, AT ,HASH,DLR ,PERC,CIRC,AMPR,ASTR,LPRN,RPRN,UNDS,PLUS,
  //|----`----`----`----`----`----|----`----`----`----`----`----`----|
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/keymaps/default/keymap.c]: Merge Conflict

File: keyboards/fourier/readme.md [hunk 8]
```diff
@@ -1,7 +1,11 @@
 Fourier
 ========
 
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/readme.md]: Merge conflict

File: keyboards/fourier/readme.md [hunk 9]
```diff
@@ -17,4 +21,8 @@ Example of flashing this keyboard:
 
 See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
 
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/readme.md]: Merge Conflict

File: keyboards/fourier/rev1/config.h [hunk 10]
```diff
@@ -41,6 +41,12 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 /* define if matrix has ghost */
 //#define MATRIX_HAS_GHOST
 
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/rev1/config.h]: Merge Conflict

File: keyboards/fourier/rev1/config.h [hunk 11]
```diff
@@ -62,6 +68,13 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define ws2812_PORTREG  PORTD
 #define ws2812_DDRREG   DDRD
 
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/rev1/config.h]: merge conflict

File: keyboards/fourier/rev1/rev1.h [hunk 12]
```diff
@@ -18,13 +18,21 @@
 	LA1, LA2, LA3, LA4, LA5, LA6, RA1, RA2, RA3, RA4, RA5, RA6, RA7, \
 	LB1, LB2, LB3, LB4, LB5, LB6, RB1, RB2, RB3, RB4, RB5,      RB7, \
 	LC1, LC2, LC3, LC4, LC5, LC6, RC1,      RC3, RC4, RC5, RC6, RC7, \
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/rev1/rev1.h]: merge conflict

File: keyboards/fourier/rev1/rev1.h [hunk 13]
```diff
@@ -18,13 +18,21 @@
...
+	LD1, LD2, LD3,      LD5,      RD1,           RD4, RD5, RD6, RD7 \
+=======
 	LD1, LD2, LD3, LD4, LD5,      RD1,           RD4, RD5, RD6, RD7 \
+>>>>>>> 19a1fbaca2137a9d4cd529e363039a612a4f941d
 	) \
 	{ \
 		{ LA1, LA2, LA3, LA4, LA5, LA6, KC_NO}, \
 		{ LB1, LB2, LB3, LB4, LB5, LB6, KC_NO}, \
 		{ LC1, LC2, LC3, LC4, LC5, LC6, KC_NO}, \
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/rev1/rev1.h]: Merge conflict

File: keyboards/fourier/rules.mk [hunk 14]
```diff
@@ -1,7 +1,12 @@
 SRC += matrix.c \
 	   i2c.c \
 	   split_util.c \
+<<<<<<< HEAD
```
Comment (drashna) [inline keyboards/fourier/rules.mk]: Merge conflict

**Comment by drashna** [comment]
It looks like you've removed the fourier keyboard files.

**Comment by Valgrahf** [comment]
I still had the original keyboards/fourier files from nooges original prototype repo, thus the difference between the two sets of files. Git noob that I am, I finally figured out what I needed to do to fix the issue.

**Comment by Valgrahf** [comment]
Revision to the PCB from rev 1.0 also changed the other things, it seems. Further reworking required.

**Pull request closed without merging**